enterprise/coderd/dynamicparameters_test.go

```diff
@@ -127,3 +415,27 @@ func TestDynamicParameterTemplate(t *testing.T) {
 	coderdtest.AssertParameter(t, "thing", resp.Parameters).
 		Exists().Value("banana").Options("banana", "ocean", "sky")
 }
+
+func assertWorkspaceBuildParameters(ctx context.Context, t *testing.T, client *codersdk.Client, buildID uuid.UUID, values map[string]string) {
+	t.Helper()
+
+	params, err := client.WorkspaceBuildParameters(ctx, buildID)
+	require.NoError(t, err)
+
+	for name, value := range values {
+		param, ok := slice.Find(params, func(parameter codersdk.WorkspaceBuildParameter) bool {
+			return parameter.Name == name
+		})
+		if !ok {
+			assert.Failf(t, "parameter not found", "expected parameter %q to exist with value %q", name, value)
+			continue
+		}
+		require.Equalf(t, value, param.Value, "parameter %q should have value %q", name, value)
```

Comment:
For consistency, assert here as well? 
```suggestion
		assert.Equalf(t, value, param.Value, "parameter %q should have value %q", name, value)
```